src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -162,8 +162,11 @@ private static JsonObject createLoanContext(
       } else {
         int renewalLimit = loanPolicy.getRenewalLimit();
         int renewalsRemaining = renewalLimit - loan.getRenewalCount();
```

Comment:
Ensuring this cannot be negative, could be done in a single line:
```suggestion
        int renewalsRemaining = max(renewalLimit - loan.getRenewalCount(), 0);
```

Reply:
Thanks for the suggestion. Done